### PR TITLE
Added documentation for facet view table

### PR DESCRIPTION
**Pull request by saschabartsch**
**custom-view-for-table-widget.md**
- Actual documentation file
**Media-Folder**
- Added necessary screenshots
**mkdocs.yml**
- Added navigation entry

**Description**
Added a documentation for the new feature **Custom views for table widget**

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -191,6 +191,7 @@ nav:
         - "Method: RASCI vs. RACI in detail content": admin/features/process-design/raci_instead_rasci/raci-instead-of-rasci.md
         - "Method: Changed evaluation groups for processes": admin/features/process-design/changed-evaluation-groups-processes/changed-evaluation-groups-processes.md
         - "Fact Sheets: Global related objects": admin/features/process-design/fact_sheet_related-global-objects/fact-sheet_related-global-objects.md
+        - "Fact Sheets: Custom views for table widget": admin/features/process-design/custom-views-for-table-widget/custom-views-for-table-widget.md
```
Comment (thomashauser) [inline mkdocs.yml:194]: Move it to general design

**Review by thomashauser — CHANGES_REQUESTED** [review]
Separate documentation for consolidation of 'create dropdown value' customizations is missing. Do we create a separate PR for that?

**Comment by saschabartsch** [comment]
> 
> 
> Separate documentation for consolidation of 'create dropdown value' customizations is missing. Do we create a separate PR for that?

I don't think we need a seperate PR for that since it was part of the same PBI. I am just not familiar with these changes. You can show me what changed and I will create another entry in the documentation

File: mkdocs.yml (hunk 1 above)
Comment (saschabartsch) [reply mkdocs.yml]: done

File: docs/admin/features/general-design/consolidation-of-extended-configurations/consolidation-of-extended-configurations.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+# Consolidation of extended configurations
+
+## Description
+With this feature activated, you have the possibility to consolidate two given customizations of the same type to one customization containing the settings of both customizations.
+
+An exemplary use case for this feature is the usage of two identical **custom dropdown values** in two different **dropdown attributes**. You can now select one of the custom **values** as a **'master'** attribute with the other custom **value** as a **'slave'**. Consolidating these to values will result in the **master** overtaking the **attribute types** of the **slave** while the latter is deleted.
```
[the remark this answers is not in the document]
Comment (saschabartsch) [reply docs/admin/features/general-design/consolidation-of-extended-configurations/consolidation-of-extended-configurations.md]: I fixed the wording in the description and added a small paragraph with a note that it is also possible to do the example with multiple slaves and one master

[the remark this answers is not in the document]
Comment (saschabartsch) [reply docs/admin/features/general-design/consolidation-of-extended-configurations/consolidation-of-extended-configurations.md]: fixed

**Review by thomashauser — APPROVED** [review]

**Pull request merged**